packages/core/src/utilities.ts

```diff
@@ -34,6 +34,47 @@ export function extractStyles(style: StyleProp<any>) {
   return { viewStyles, textStyles };
 }
 
+export const borderStyleNames = [
+  "borderRadius",
+  "borderTopLeftRadius",
+  "borderTopRightRadius",
+  "borderBottomRightRadius",
+  "borderBottomLeftRadius",
+  "borderTopWidth",
+  "borderRightWidth",
+  "borderBottomWidth",
+  "borderLeftWidth",
+  "borderColor",
+  "borderStyle",
+];
+
+export const marginStyleNames = [
+  "marginLeft",
```

Comment:
marginHorizontal/marginVertical?

Reply:
@peterpme added all margin and border styles.